chrome/content/zotero/zoteroPane.js

```diff
@@ -515,6 +515,18 @@ var ZoteroPane = new function()
 					this.highlightTimer = null;
 				}
 				ZoteroPane_Local.collectionsView.setHighlightedRows();
+				return;
+			} else if (event.keyCode == event.DOM_VK_BACK_QUOTE) {
```

Comment:
Consensus? I like backtick myself too, easy to touch-locate and useful for multi-hand scenarios, if you have the right hand on the mouse/touchpad/arrow keys.